projects/etos_suite_runner/src/etos_suite_runner/__main__.py

```diff
@@ -27,12 +31,39 @@
 
 def main():
     """Entry point allowing external calls."""
-    esr = ESR()
+    etos = ETOS("ETOS Suite Runner", os.getenv("SOURCE_HOST"), "ETOS Suite Runner")
+    etos.config.set("results", [])
+    esr = ESR(etos)
     try:
         esr.run()  # Blocking
+        results = etos.config.get("results") or []
+        result = None
+        for suite_result in results:
+            if suite_result.get("verdict") == "FAILED":
+                result = suite_result
+                break
+            if suite_result.get("verdict") == "INCONCLUSIVE":
+                result = suite_result
+        if len(results) == 0:
+            result = {
+                "conclusion": "Inconclusive",
+                "verdict": "Inconclusive",
+                "description": "Got no results from ESR"
+            }
+        elif result is None:
+            # No suite failed, so lets just pick the first result
+            result = results[0]
+        # Convert, for example, INCONCLUSIVE to Inconclusive to match the controller result struct
+        # TODO Move the result struct to ETOS library and do this conversion on creation
+        result["conclusion"] = f"{result['conclusion'][0].upper()}{result['conclusion'][1:].lower()}"
```

Comment:
could be:

result['conclusion'].title()  

instead of:

f"{result['conclusion'][0].upper()}{result['conclusion'][1:].lower()}"